scripts/minute_into_hour_series.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+
+import pandas as pd
+
+directory = Path("building_dialouge_webapp/data/profiles")
+date = "2023-01-01"  # Dummy date for timestamp column
+
+for filename in directory:
+    if filename.endswith(".csv"):
+        file_path = filename
+
+        # Read the first row to get column names
+        with file_path.open(encoding="utf-8") as f:
+            first_line = f.readline().strip()
+            column_names = first_line.split(",")
+
+        # Read the CSV again, skipping the first row
+        df_original = pd.read_csv(file_path, skiprows=1, names=column_names)
```

Comment:
FYI: Header can be handled by pandas as well by parameter `header`. Normally, `read_csv` should guess it by default. But I'm fine with your solution - thanks!